setup.py

```diff
@@ -34,7 +34,7 @@
         ),
     ],
     install_requires=[
-        "policyengine-core",
+        "policyengine-core >= 3.6.3",
```

Comment:
by convention - spaces might work, not sure
```suggestion
        "policyengine-core>=3.6.3",
```